### PR TITLE
[AMP] Fix upload_date and timestamp extraction

### PR DESCRIPTION
**Pull request by adrianheine**
### Before submitting a *pull request* make sure you have:
- [x] [Searched](https://github.com/ytdl-org/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Read [adding new extractor tutorial](https://github.com/ytdl-org/youtube-dl#adding-support-for-a-new-site)
- [x] Read [youtube-dl coding conventions](https://github.com/ytdl-org/youtube-dl#youtube-dl-coding-conventions) and adjusted the code to meet them
- [x] Covered the code with tests (note that PRs without tests will be REJECTED)
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Bug fix
- [ ] Improvement
- [ ] New extractor
- [ ] New feature

### CODE REVIEW

File: youtube_dl/extractor/amp.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
     mimetype2ext,
     parse_iso8601,
     url_or_none,
+    unified_timestamp,
```
Comment (remitamine) [inline youtube_dl/extractor/amp.py:12]: Alphabetic.

**Review by remitamine — CHANGES_REQUESTED** [review]
`BleacherReportCMSIE` test is also affected by this change.

File: youtube_dl/extractor/amp.py (hunk 1 above)
Comment (adrianheine) [reply youtube_dl/extractor/amp.py:12]: Oh I tried

**Comment by adrianheine** [comment]
> `BleacherReportCMSIE` test is also affected by this change.

That video seems to not be available anymore anyway.

**Comment by remitamine** [comment]
> > `BleacherReportCMSIE` test is also affected by this change.
> 
> That video seems to not be available anymore anyway.

the extractor does not rely on the webpage.

**Comment by adrianheine** [comment]
> the extractor does not rely on the webpage.

You're right, thanks for your diligence.

**Pull request merged**